File: Java/DynamicArray.java
import java.util.*;
public class DynamicArray{

   public static void main (String[] args){
    int n, arr[];
    Scanner sc = new Scanner(System.in);
    System.out.print("Enter the size of array: ");
    n= sc.nextInt();
    arr=new int[n];

    System.out.printf("Enter %d values below:\n",n);

    for(int i=0; i<n; i++)
    {
        arr[i]=sc.nextInt();
    }
    System.out.println(" The elements of the array are: ");

    for(int i=0; i<n; i++)
    {
        System.out.println(arr[i]);
    }

    System.out.println("The sorted array is: ");
    Arrays.sort(arr);

    for(int i=0; i<n; i++)
    {
        System.out.println(arr[i]);
    }
    sc.close();
   } 
}

File: CPP/DynamicArray.cpp
#include<iostream>
using namespace std;
int main()
{
    int n, *arr;
    cout<<"Enter the size of the array: ";
    cin>>n;

    arr = new int[n];
    cout<<"Enter "<<n<<"value below : "<<endl;
    for(int i=0; i<n; i++)
    {
      cin>>arr[i];
    }

    cout<<"The elements are the array are:"<<endl;
    for(int i=0; i<n; i++)
    {
        cout<<arr[i]<<endl;
    }
    return 0;
}